packages/story-editor/src/components/library/libraryProvider.js

```diff
@@ -44,7 +44,13 @@ const LIBRARY_TAB_IDS = new Set(
 );
 
 function LibraryProvider({ children }) {
-  const [tab, setTab] = useState(MEDIA.id);
+  const { showMedia3p } = useConfig();
+  const {
+    actions: { getMedia },
+  } = useAPI();
+  const showMedia = Boolean(getMedia); // Do not show media tab if getMedia api callback is not provided.
+  const defaultCurrentTabId = showMedia ? MEDIA.id : MEDIA3P.id;
+  const [tab, setTab] = useState(defaultCurrentTabId);
```

Comment:
This won't work if `showMedia3p` is false.

Might be easier to just default to `const [tab, setTab] = useState(tabs[0])`

Reply:
Good idea, updated, thanks.